olvg-luscii-protocol-v01.md

```diff
@@ -289,6 +291,69 @@ Figure 4:Introduction survey 1                           |  Figure 5:Introductio
 :----------------------------------------------------------------:|:--------------------------------------------------:
 Figure 7:Healthcare progression admin                             |  Figure 8:Participant app
 
+
+---
+
+## Updates
+This document was initially concieved on 20-03-2020; as the tele-monitoring application we use for the DOC-COV study is in active use to help reduce the burden on the healthcare system in various places this protocol is subject to change. In this section of this document we try keep track -- as much as possible -- of changes that occur over time. On each pupdate we update both the recruitment efforts and the alert rules (and any other things that we feel are methodologically worth mentioning).
```

Comment:
**On each `pupdate` we update both the recruitment efforts** is `update`?

Reply:
Hi, yeah, that's what I was intending; effectively to have a part of the document that is augmented each time we want to expland on the documentation. Does that work?